src/commands/runDraftTool/runDraftWorkflow.ts

```diff
@@ -0,0 +1,471 @@
+}
+
+class PromptGitHubBranchSelection extends AzureWizardPromptStep<WizardContext> {
+   public async prompt(wizardContext: WizardContext): Promise<void> {
+      if (wizardContext.destination === undefined) {
+         throw Error('Git Directory is Undefined');
+      }
+      let branches = await getBranches(wizardContext.destination);
+      branches = branches.filter(
+         (b) => b.remote !== undefined && b.remote !== ''
+      );
+
+      const branchToItem = (b: Branch): vscode.QuickPickItem => ({
+         label: b.name?.replace(b.remote + '/', '') || ''
+      });
+      const branchOptions = branches.map(branchToItem);
+
+      const selectedBranch = await wizardContext.ui.showQuickPick(
+         branchOptions,
+         {
+            ignoreFocusOut,
+            stepName: 'GitHub Branch',
+            placeHolder: 'GitHub Branch'
+         }
+      );
+
+      wizardContext.branch = selectedBranch.label;
+   }
+
+   public shouldPrompt(wizardContext: WizardContext): boolean {
+      return true;
+   }
+}
+
+class ExecuteDraftWorkflow extends AzureWizardExecuteStep<WizardContext> {
+   public priority: number = 2;
+
+   public async execute(
+      wizardContext: WizardContext,
+      progress: vscode.Progress<{
+         message?: string | undefined;
+         increment?: number | undefined;
+      }>
+   ): Promise<void> {
+      const {
+         destination,
+         clusterSubscription,
+         clusterResourceGroup,
+         cluster,
+         acrSubscription,
+         acrResourceGroup,
+         registry,
+         acrRepository,
+         branch
+      } = wizardContext;
+      if (destination === undefined) {
+         throw Error('Output folder is undefined');
+      }
+      if (clusterSubscription === undefined) {
+         throw Error('AKS Subscription is undefined');
+      }
+      if (clusterResourceGroup === undefined) {
+         throw Error('AKS Resource Group is undefined');
+      }
+      if (cluster === undefined) {
+         throw Error('AKS Cluster is undefined');
+      }
+      if (acrResourceGroup === undefined) {
+         throw Error('AKS Resource Group is undefined');
+      }
+      if (registry === undefined) {
+         throw Error('ACR Registry is undefined');
+      }
+      if (acrRepository === undefined) {
+         throw Error('ACR Repository is undefined');
+      }
+      if (branch === undefined) {
+         throw Error('Branch is undefined');
+      }
+
+      const generateWorkflowCmd = buildGenerateWorkflowCommand(
+         destination.fsPath,
+         cluster.managedCluster.name || '',
+         clusterResourceGroup.resourceGroup.name || '',
+         registry.registry.name || '',
+         acrRepository.repositoryName,
+         branch
+      );
+      const [success, err] = await runDraftCommand(generateWorkflowCmd);
+      const isSuccess = err?.length === 0 && success?.length !== 0;
+      if (!isSuccess) {
+         throw Error(`Draft command failed: ${err}`);
+      }
```

Comment:
Just thought about this but we should tell the user that we succeeded somehow. On similar commands we open the generated files. It would make sense to do that here too.

Reply:
added the ExecuteOpenWorkflowFile command to open after the workflow is generated